tests/tools/nnpackage_run/src/args.cc

```diff
@@ -16,11 +16,46 @@
 
 #include "args.h"
 
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/foreach.hpp>
```

Comment:
@hyunsik-yoon I and other members (e.g. @wateret) would like to make `nnpackage_run` boost-free. Thus, I removed `boost::filesystem` dependency. `program_options` are the only dependency. I would like to implement without further `boost`.

Reply:
What about using jsoncpp? we already have it.

Reply:
OK. I will fix it using jsoncpp.